src/js/blue-button/components/GlossaryModal.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Modal from '../../common/components/Modal';
+
+class GlossaryModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleCloseModal(event) {
+    event.preventDefault();
+    this.props.onCloseModal();
+  }
+
+  render() {
+    if (this.props.isVisible) {
```

Comment:
Can this be removed? This seems redundant with the visible property of the modal